### PR TITLE
Michael/ir 5285 scene preview thumbnail

### PR DESCRIPTION
**Pull request by MbfloydIR**
## Summary
https://tsu.atlassian.net/browse/IR-5285
## References
closes #_insert number here_

### CODE REVIEW

File: packages/ui/src/components/editor/properties/scene/previewCamera/index.tsx [hunk 1]
```diff
@@ -58,18 +64,58 @@ const scene = new Scene()
...
   const { t } = useTranslation()
   const [bufferUrl, setBufferUrl] = useState<string>('')
+  const [blob, setBlob] = useState<Blob>()
   const transformComponent = useComponent(Engine.instance.cameraEntity, TransformComponent)
 
   const onSetFromViewport = () => {
     const { position, rotation } = getComponent(Engine.instance.cameraEntity, TransformComponent)
     const transform = getComponent(props.entity, TransformComponent)
+    const camera = getComponent(props.entity, ScenePreviewCameraComponent).camera
+    camera.position.copy(position)
```
Comment (DanielBelmes) [inline packages/ui/src/components/editor/properties/scene/previewCamera/index.tsx]: @HexaField So this is a piece I had to help with. It seems like commitTransformSave wasn't updating the scene preview camera fast enough. So I asked Michael to update the three object directly. Are there alternative ways to get this done?

Comment (DanielBelmes) [reply packages/ui/src/components/editor/properties/scene/previewCamera/index.tsx]: For context there's a button to set scene preview to the viewport and then update the screenshot. It sends a transform update but the screenshot happens instantly after which I think is just happening before SnapshotSystem, TransformSystem, etc....

Comment (DanielBelmes) [reply packages/ui/src/components/editor/properties/scene/previewCamera/index.tsx]: Maybe a useEffect is the answer detecting if the ScenePreviewCamera has moved.

Comment (HexaField) [reply packages/ui/src/components/editor/properties/scene/previewCamera/index.tsx:74]: ah this makes sense. yeah just update the position and then call `computeTransformMatrix`

Comment (HexaField) [reply packages/ui/src/components/editor/properties/scene/previewCamera/index.tsx]: sorry just getting back to this now

**Review by HexaField — APPROVED** [review]
i merged dev in, which includes rahul's fixes, and this works really well now :)

**Pull request merged**